pages/__tests__/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from "@testing-library/react";
+
+import SectionTextButtonImage from "@/components/SectionTextButtonImage/SectionTextButtonImage";
+
+describe(SectionTextButtonImage.name, () => {
```

Comment:
I'm curious why are you copying the test from the other component here? if the component has already been tested in this constellation there is no need to do it again.